src/patternfly/utilities/BoxShadow/box-shadow.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="{{#if box-shadow--modifier}} {{box-shadow--modifier}}{{/if}}"
```

Comment:
Since `{{box-shadow--modifier}}` is the only class, you can removing the leading space before the output.